pkg/cmd/stack/destroy_test.go

```diff
@@ -277,3 +278,74 @@ func Test_destroyJobs(t *testing.T) {
 		})
 	}
 }
+
+func Test_onlyDeployVolumesFromServicesToDeploy(t *testing.T) {
```

Comment:
shouldn't this be on stack/deploy_test.go

Reply:
Yes, thanks. I misread